### PR TITLE
Feature/AndesMessage Thumbnail

### PR DESCRIPTION
**Pull request by javff**
## Description
Add thumbnail variant in AndesMessage

## RFC Link
https://docs.google.com/document/d/1PjuVsZUQZQAzsb0p8x1XC4AFHqPx0B3Hgd8Ipb8jL1Q/edit#

## Screenshots / GIFs

| default | with-actions  | with-bullets |
| --- | --- | --- | 
deafult | with-actions | bullets |

| examples |
| --- | 
simulator_screenshot_CF6380F5-EACA-4580-8329-FA3E03323E9C

## Checks
Are you sure you followed all those steps? If so, repeat after me "I solemnly swear that ...":
   - [x] I have coded an example inside the Demo App.
   - [x] I have added proper tests.
   - [x] I have modified the Changelog.md file.
   - [ ] I have updated GitHub wiki.
   - [ ] I have the explicit approval of one or more members of the UX Team.
   - [x] I have notify Andes team for PR.
   - [x] I have checked that this code is ObjC compliant.
   - [x] I have checked that all the new public enums conform to AndesEnumStringConvertible.
## Change type
This is easy, let us know what this code is about:
   - [ ] This code adds a new component
   - [ ] This code includes improvements to an existent component
   - [ ] This code improves or modifies ONLY the Demo App.

### CODE REVIEW

File: Example/AndesUI/Messages/Views/MessageObjCViewController.m [hunk 1]
```diff
@@ -75,6 +76,14 @@ - (void)setup {
     
     [_stackView addArrangedSubview:view];
     [view.widthAnchor constraintEqualToAnchor:_stackView.widthAnchor].active = YES;
+    [self setThumbnails];
+}
```
Comment (dabeltran95) [inline Example/AndesUI/Messages/Views/MessageObjCViewController.m:80]:
```suggestion
    }
```

Comment (dabeltran95) [reply Example/AndesUI/Messages/Views/MessageObjCViewController.m:80]: error de identacion

File: Example/AndesUI/Messages/Views/MessageObjCViewController.m [hunk 2]
```diff
@@ -75,6 +76,14 @@ - (void)setup {
...
+
+-(void) setThumbnails {
+    for(AndesMessage *message in self.andesMessagesWithThumbnail) {
+        message.thumbnail = [UIImage imageNamed:@"avatar-example"];
+
```
[the remark this answers is not in the document]
Comment (javff) [reply Example/AndesUI/Messages/Views/MessageObjCViewController.m:85]: buena ahi lo acomodo.

File: Example/AndesUI/Messages/Views/MessageObjCViewController.m (hunk 1 above)
Comment (javff) [reply Example/AndesUI/Messages/Views/MessageObjCViewController.m:80]: el espaciado en este caso esta bien, este archivo es .m y es el nivel de identación de los métodos.

**Review by dabeltran95 — APPROVED** [review]

**Review by dabeltran95 — CHANGES_REQUESTED** [review]
Olvidaste modificar el CHANGELOG.md añadiendo lo que hiciste :D

**Review by dabeltran95 — APPROVED** [review]

**Pull request merged**